tests.js

```diff
@@ -1,12 +1,22 @@
+
 console.lol('This message was printed with console.lol');
 
+
 console.rainbow('This message was printed with console.rainbow');
 console.rainbow([1,2,4,'testing an array!']);
 console.rainbow('testing', 'multiple rainbow args', [1,2,3,4]);
-console.rainbow('prints obje cts', {lol:'pls',ok:'naj'})
+console.rainbow('prints objects', {lol:'pls',ok:'naj'});
```

Comment:
Thanks for tidying up the code! Next time, I would give them their own commit - that way if we ever needed the `console.boo` changes by themselves for whatever reason, we don't also get all the cleaning changes.